src/Chapter1.hs

```diff
@@ -641,8 +667,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit:: Int -> Int
+firstDigit n
+  | n `div` 10 == 0 = n
```

Comment:
I see, that for a negative number like -19, this implementation will return the negative number itself instead of the last digit because the first check n < 10 will always succeed for negative numbers.